Framework/ScriptRepository/src/ScriptRepositoryImpl.cpp

```diff
@@ -1311,31 +1311,29 @@ std::string ScriptRepositoryImpl::ignorePatterns() {
 int ScriptRepositoryImpl::setAutoUpdate(const std::string &input_path,
                                         bool option) {
   ensureValidRepository();
-  std::string path = convertPath(input_path);
-  std::vector<std::string> files_to_update;
+  const std::string path = convertPath(input_path);
+  std::vector<std::string> filesToUpdate;
   for (auto it = repo.rbegin(); it != repo.rend(); ++it) {
     // for every entry, it takes the path and RepositoryEntry
-    std::string entry_path = it->first;
+    std::string entryPath = it->first;
     RepositoryEntry &entry = it->second;
-    if (entry_path.compare(0, path.size(), path) == 0 &&
+    if (entryPath.compare(0, path.size(), path) == 0 &&
         entry.status != REMOTE_ONLY && entry.status != LOCAL_ONLY)
-      files_to_update.push_back(entry_path);
+      filesToUpdate.push_back(entryPath);
   }
 
-  // g_log.debug() << "SetAutoUpdate... begin\n";
   try {
-    for (auto &path : files_to_update) {
-      RepositoryEntry &entry = repo.at(path);
+    for (const auto &fileToUpdate : filesToUpdate) {
+      RepositoryEntry &entry = repo.at(fileToUpdate);
       entry.auto_update = option;
-      updateLocalJson(path, entry); // TODO: update local json without opening
-                                    // and close file many times
+      updateLocalJson(fileToUpdate, entry);
+      // TODO: update local json without opening and closing file many times
```

Comment:
This TODO's been there for 4 years, maybe it can be moved into an issue and removed from the code?

Reply:
Done. See issue #25398.